pkg/actuators/machine/actuator.go

```diff
@@ -24,13 +24,12 @@ import (
 	"github.com/aws/aws-sdk-go/aws"
 	"github.com/aws/aws-sdk-go/service/ec2"
 	"github.com/golang/glog"
+	clustererror "github.com/openshift/cluster-api/pkg/controller/error"
```

Comment:
openshift/cluster-api is deprecated
Please use anything from machine-api-operator